specs/DotNetProjectFile.Analyzers.Specs/FluentAssertions/FindsExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿using CodeAnalysis.TestTools.Contexts;
+using Microsoft.CodeAnalysis;
+
+namespace FluentAssertions;
+
+internal static class FindsExtensions
+{
+    public static void HasNoIssues(this ProjectAnalyzerVerifyContext context)
+        => context.HasIssues();
```

Comment:
This is a bit confusing, it has no issues when it has issues without parameter?

Reply:
`HasIssues` has `param Issue[] issues`, so without params is equivalent to no issues.

Reply:
I've tried to make that easier to grasp.